doc/build-windows.md

```diff
@@ -59,21 +59,12 @@ Note that for WSL the Bitcoin Core source path MUST be somewhere in the default
 example /usr/src/bitcoin, AND not under /mnt/d/. If this is not the case the dependency autoconf scripts will fail.
 This means you cannot use a directory that is located directly on the host Windows file system to perform the build.
 
-Additional WSL Note: WSL support for [launching Win32 applications](https://learn.microsoft.com/en-us/archive/blogs/wsl/windows-and-ubuntu-interoperability#launching-win32-applications-from-within-wsl)
-results in configure scripts being able to execute Windows Portable Executable files. This can cause
-unexpected behaviour during the build, such as Win32 error dialogs for missing libraries. The recommended approach
-is to temporarily disable WSL support for Win32 applications.
-
 Build using:
 
-    PATH=$(echo "$PATH" | sed -e 's/:\/mnt.*//g') # strip out problematic Windows %PATH% imported var
-    sudo bash -c "echo 0 > /proc/sys/fs/binfmt_misc/status" # Disable WSL support for Win32 applications.
```

Comment:
Aren't the above 2 lines still needed if building with cmake on WSL?

Reply:
No, those 2 lines are not needed. I have tested this on Windows 11 using WSL and confirmed that the build process works correctly without them.

I hope you'll be able to build as well.

Reply:
I've additionally verified build logs for the `bdb` and `sqlite` packages. Enabling or disabling `binfmt_misc` makes no difference.

Reply:
If this is the case, then it should just be removed in master? Given this isn't related to CMake.

Reply:
> If this is the case, then it should just be removed in master? Given this isn't related to CMake.

The root `configure` script in the master branch is still affected:
```diff
$ diff -u ~/configure_binfmt_disabled ~/configure_binfmt_enabled
--- /home/hebasto/configure_binfmt_disabled     2024-08-06 15:52:36.633707879 +0100
+++ /home/hebasto/configure_binfmt_enabled      2024-08-06 15:50:27.099730962 +0100
@@ -15,7 +15,7 @@
 checking whether the C++ compiler works... yes
 checking for C++ compiler default output file name... a.exe
 checking for suffix of executables... .exe
-checking whether we are cross compiling... yes
+checking whether we are cross compiling... no
 checking for suffix of object files... o
 checking whether the compiler supports GNU C++... yes
 checking whether x86_64-w64-mingw32-g++-posix accepts -g... yes
@@ -50,7 +50,6 @@
 checking for /usr/bin/x86_64-w64-mingw32-ld option to reload object files... -r
 checking for x86_64-w64-mingw32-file... no
 checking for file... file
-configure: WARNING: using cross tools not prefixed with host triplet
 checking for x86_64-w64-mingw32-objdump... x86_64-w64-mingw32-objdump
 checking how to recognize dependent libraries... (cached) pass_all
 checking for x86_64-w64-mingw32-dlltool... x86_64-w64-mingw32-dlltool
@@ -351,7 +350,7 @@
 checking whether the C compiler works... yes
 checking for C compiler default output file name... a.exe
 checking for suffix of executables... .exe
-checking whether we are cross compiling... yes
+checking whether we are cross compiling... no
 checking for suffix of object files... o
 checking whether the compiler supports GNU C... yes
 checking whether x86_64-w64-mingw32-gcc accepts -g... yes
@@ -378,7 +377,6 @@
 checking for /usr/bin/x86_64-w64-mingw32-ld option to reload object files... -r
 checking for x86_64-w64-mingw32-file... no
 checking for file... file
-configure: WARNING: using cross tools not prefixed with host triplet
 checking for x86_64-w64-mingw32-objdump... x86_64-w64-mingw32-objdump
 checking how to recognize dependent libraries... file_magic ^x86 archive import|^x86 DLL
 checking for x86_64-w64-mingw32-dlltool... x86_64-w64-mingw32-dlltool
```

Reply:
> If this is the case, then it should just be removed in master? Given this isn't related to CMake.

Please see https://github.com/bitcoin/bitcoin/pull/30597 for the first removed line.